cmake/modules/RootConfiguration.cmake

```diff
@@ -560,6 +560,7 @@ string(REPLACE "c++11" "cxx11" features ${features}) # change the name of the c+
 set(configfeatures ${features})
 set(configargs ${ROOT_CONFIGARGS})
 set(configoptions ${ROOT_CONFIGARGS})
+set(configstd ${CMAKE_CXX${CMAKE_CXX_STANDARD}_STANDARD_COMPILE_OPTION})
```

Comment:
Where does this variable come from? I couldn't find it in the documentation.

Reply:
I've asked about it here: <https://gitlab.kitware.com/cmake/cmake/issues/18910>

Reply:
You can look through CMake to see them in the meantime, here's the compiler file for CMake 3.1 and g++: <https://gitlab.kitware.com/cmake/cmake/blob/v3.1.0/Modules/Compiler/GNU-CXX.cmake>